Source/DafnyLanguageServer/Language/Symbols/DafnyLangSymbolResolver.cs

```diff
@@ -129,13 +142,13 @@ private class SymbolDeclarationResolver {
           default:
             // TODO The last missing member is AmbiguousMemberDecl which is created by the resolver.
             //      When is this class exactly used?
-            _logger.LogDebug("encountered unknown class member declaration {DeclarationType}", memberDeclaration.GetType());
+            logger.LogDebug("encountered unknown member declaration {DeclarationType}", memberDeclaration.GetType());
             return null;
         }
       }
 
       private FunctionSymbol ProcessFunction(Symbol scope, Function function) {
-        _cancellationToken.ThrowIfCancellationRequested();
+        cancellationToken.ThrowIfCancellationRequested();
```

Comment:
It might be easier to move this throw and the ones for method and field to line 124. In general, I would expect `cancellationToken.ThrowIfCancellationRequested();` at the start of loop bodies.